mongo_fix/mongo_fix.go

```diff
@@ -0,0 +1,42 @@
+// pmm-agent
+// Copyright 2019 Percona LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package mongo_fix
+
+import (
+	"net/url"
+
+	"github.com/pkg/errors"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+// ClientForDSN applies URI to Client.
+func ClientForDSN(dsn string) (*options.ClientOptions, error) {
```

Comment:
probably we should call it `ClientOptionsForDSN` or include `mongo.Connect` to this function

Reply:
yeah, `ClientOptions` is more correct